tests/covalent_dispatcher_tests/_core/data_manager_test.py

```diff
@@ -113,9 +169,23 @@ def workflow(x):
     assert sub_result_object._root_dispatch_id == result_object.dispatch_id
 
 
-@pytest.mark.parametrize("node_status", [Result.COMPLETED, Result.FAILED, Result.CANCELLED])
+@pytest.mark.parametrize(
```

Comment:
You can automatically have a matrix of parameters if you stack two (or 3) different decorators on top of one another instead of manually copying the same thing multiple times.